src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -107,18 +107,26 @@
 
 (defn rename-color-id-in-data
   [data]
-  (map (fn [item]
-         (update item
-                 :accounts
-                 (fn [accounts]
-                   (map (fn [account]
-                          (let [renamed-account (set/rename-keys account
-                                                                 {:colorId :customization-color})]
-                            (if (contains? account :colorId)
-                              renamed-account
-                              (assoc renamed-account :customization-color :blue))))
-                        accounts))))
-       data))
+  (let [sorted-data (sort-by #(if (some (fn [account]
```

Comment:
hmm, we must already be doing the colorId conversion elsewhere in the app? do we really need to re make this logic again?
cc @smohamedjavid, @yqrashawn, @ibrkhalil

Reply:
@OmarBasem 
Additionally to what Jamie said, what is `data`? can't we give a better name? It was hard for me to realize what is this.

Just as extra feedback, In this function:

```clojure
(fn [account]
  (let [renamed-account (set/rename-keys account
                                         {:colorId :customization-color})]
    (if (and (contains? account :colorId) (seq (get account :colorId)))
      (assoc renamed-account :customization-color (keyword (get account :colorId)))
      (assoc renamed-account :customization-color :blue))))
```

We `rename-keys` and then we `assoc` the  renamed key, seems a bit unnecessary to rename then.
I was wondering if this function might work in the same way:

```clojure
(fn [{:keys [colorId] :as account}]
  (assoc account :customization-color (if (seq colorId) ;; if seq is not contained or is nil, it'll return false
                                        (keyword colorId)
                                        :blue)))
                                        ;; we can also add a `dissoc` to `:colorID` if needed
```

Reply:
@J-Son89 - Yes, I guess we can use this existing method from `data-store`.

https://github.com/status-im/status-mobile/blob/7d6b2ac87a4dfd869cebf1be01d252304f17b959/src/status_im/contexts/wallet/data_store.cljs#L49-L53

Reply:
> Just as extra feedback, In this function

@ulisesmac  Thanks for the suggestion

Reply:
@J-Son89 @smohamedjavid the function `rpc->accounts` doesn't do any renaming, it keywordize the value of colorId and assigns a default. Btw, this function does more than just renaming colorId, it also sorts the keypairs. I have taken the sort part out of the function to make it more readable.